### PR TITLE
Add/Update some enums and structs

### PR DESCRIPTION
**Pull request by scribam**

### CODE REVIEW

File: include/psp2/appmgr.h [hunk 1]
```diff
@@ -20,7 +20,8 @@ typedef enum SceAppMgrErrorCode {
 	SCE_APPMGR_ERROR_NULL_POINTER       = 0x80802016, //!< NULL pointer
 	SCE_APPMGR_ERROR_INVALID            = 0x8080201A, //!< Invalid param
 	SCE_APPMGR_ERROR_INVALID_SELF_PATH  = 0x8080201E, //!< Invalid SELF path
-	SCE_APPMGR_ERROR_TOO_LONG_ARGV      = 0x8080201D  //!< argv is too long
+	SCE_APPMGR_ERROR_TOO_LONG_ARGV      = 0x8080201D, //!< argv is too long
```
Comment (d3m3vilurr) [inline include/psp2/appmgr.h]: please sort this

File: include/psp2/apputil.h [hunk 2]
```diff
@@ -31,12 +31,17 @@ typedef enum SceAppUtilErrorCode {
 	SCE_APPUTIL_ERROR_BUSY                        = 0x80100603,
 	SCE_APPUTIL_ERROR_NOT_MOUNTED                 = 0x80100604,
 	SCE_APPUTIL_ERROR_NO_PERMISSION               = 0x80100605,
+	SCE_APPUTIL_ERROR_PASSCODE_MISMATCH           = 0x80100606,
+	SCE_APPUTIL_ERROR_STACKSIZE_TOO_SHORT         = 0x801006A0,
```
Comment (d3m3vilurr) [inline include/psp2/apputil.h]: same here

File: include/psp2/message_dialog.h [hunk 3]
```diff
@@ -70,7 +79,10 @@ typedef enum SceMsgDialogButtonId {
 	SCE_MSG_DIALOG_BUTTON_ID_OK         = 1,
 	SCE_MSG_DIALOG_BUTTON_ID_YES        = 1,
 	SCE_MSG_DIALOG_BUTTON_ID_NO         = 2,
-	SCE_MSG_DIALOG_BUTTON_ID_RETRY      = 3
+	SCE_MSG_DIALOG_BUTTON_ID_RETRY      = 3,
+	SCE_MSG_DIALOG_BUTTON_ID_BUTTON1    = 1,
+	SCE_MSG_DIALOG_BUTTON_ID_BUTTON2    = 2,
+	SCE_MSG_DIALOG_BUTTON_ID_BUTTON3    = 3
```
Comment (d3m3vilurr) [inline include/psp2/message_dialog.h]: why need 1,2,3?

File: include/psp2/message_dialog.h [hunk 4]
```diff
@@ -38,6 +38,7 @@ typedef enum SceMsgDialogSystemMessageType {
 	SCE_MSG_DIALOG_SYSMSG_TYPE_NOSPACE                        = 2,
 	//! Displays "Move away from the source of interference, or adjust the compass by moving your PS Vita system as shown below."
 	SCE_MSG_DIALOG_SYSMSG_TYPE_MAGNETIC_CALIBRATION           = 3,
+	SCE_MSG_DIALOG_SYSMSG_TYPE_MIC_DISABLED                   = 4,
```
Comment (Rinnegatamante) [inline include/psp2/message_dialog.h]: Can you add message displayed for this? Same as the other messages in the enum

File: include/psp2/message_dialog.h [hunk 5]
```diff
@@ -46,14 +47,22 @@ typedef enum SceMsgDialogSystemMessageType {
 	SCE_MSG_DIALOG_SYSMSG_TYPE_NEED_MC_CONTINUE               = 7,
 	//! Displays "Cannot perform this operation. No memory card is inserted."
 	SCE_MSG_DIALOG_SYSMSG_TYPE_NEED_MC_OPERATION              = 8,
+	SCE_MSG_DIALOG_SYSMSG_TYPE_NOSPACE_CONTINUABLE            = 9,
```
Comment (Rinnegatamante) [inline include/psp2/message_dialog.h]: Same for those

File: include/psp2/message_dialog.h [hunk 6]
```diff
@@ -46,14 +47,22 @@ typedef enum SceMsgDialogSystemMessageType {
...
+	SCE_MSG_DIALOG_SYSMSG_TYPE_PATCH_FOUND                    = 13,
 	//! Displays "You must enable the microphone."
 	SCE_MSG_DIALOG_SYSMSG_TYPE_TRC_MIC_DISABLED               = 100,
 	//! Displays "You must use Wi-Fi to do this."
 	SCE_MSG_DIALOG_SYSMSG_TYPE_TRC_WIFI_REQUIRED_OPERATION    = 101,
 	//! Displays "You must use Wi-Fi to use this application."
 	SCE_MSG_DIALOG_SYSMSG_TYPE_TRC_WIFI_REQUIRED_APPLICATION  = 102,
 	//! Displays "No content is available yet."
-	SCE_MSG_DIALOG_SYSMSG_TYPE_TRC_EMPTY_STORE                = 103
+	SCE_MSG_DIALOG_SYSMSG_TYPE_TRC_EMPTY_STORE                = 103,
```
Comment (Rinnegatamante) [inline include/psp2/message_dialog.h]: And those

File: include/psp2/message_dialog.h (hunk 4 above)
Comment (scribam) [reply include/psp2/message_dialog.h]: I removed the enums as some messages can be quite long and I do not know how to format them to fit with what is expected here. I will let someone else handle it.

File: include/psp2/sysmodule.h [hunk 7]
```diff
@@ -95,6 +95,7 @@ typedef enum SceSysmoduleModuleId {
 	SCE_SYSMODULE_NEAR_DIALOG_UTIL      = 0x004A, //!< NearDialogUtil module
 	SCE_SYSMODULE_LOCATION_EXTENSION    = 0x004B, //!< LocationExt module
 	SCE_SYSMODULE_AVPLAYER              = 0x004C, //!< AVPlayer module
+    SCE_SYSMODULE_GAME_UPDATE           = 0x004D, //!< Game Update module
```
Comment (d3m3vilurr) [inline include/psp2/sysmodule.h]: wrong indent in here.

Comment (scribam) [reply include/psp2/sysmodule.h]: fixed

File: include/psp2/appmgr.h [hunk 8]
```diff
@@ -19,8 +19,9 @@ typedef enum SceAppMgrErrorCode {
 	SCE_APPMGR_ERROR_STATE              = 0x80802013, //!< Invalid state
 	SCE_APPMGR_ERROR_NULL_POINTER       = 0x80802016, //!< NULL pointer
 	SCE_APPMGR_ERROR_INVALID            = 0x8080201A, //!< Invalid param
+	SCE_APPMGR_ERROR_TOO_LONG_ARGV      = 0x8080201D, //!< argv is too long
 	SCE_APPMGR_ERROR_INVALID_SELF_PATH  = 0x8080201E, //!< Invalid SELF path
-	SCE_APPMGR_ERROR_TOO_LONG_ARGV      = 0x8080201D  //!< argv is too long
+	SCE_APPMGR_ERROR_BGM_PORT_BUSY      = 0x80803000
```
Comment (d3m3vilurr) [inline include/psp2/appmgr.h]: can you add document here?
otherwise looks good to me.

Comment (scribam) [reply include/psp2/appmgr.h]: yes, done

**Pull request merged**